app/models/progress_message.rb

```diff
@@ -3,18 +3,12 @@ class ProgressMessage < ApplicationRecord
 
   after_initialize :set_defaults, unless: :persisted?
 
-  AS_JSON_ATTRIBUTES = %w(id level message received_at).freeze
```

Comment:
I see the ID was included here - but isn't in the actual output anymore (ie not in the openapi spec). Is the openapi spec right in that regard?

Reply:
@lindgrenj6 I wondered the same, but the generated clients only expose what is declared in the spec. I ended up pulling out some `expects` that validated some of these fields since the openapi spec didn't validate them.